### PR TITLE
Feature: Download OpenAPI 

### PR DESCRIPTION
**Pull request by jwdotjs**
- Adds ability to Download OpenAPI json from workspace
- Adds unit tests with `Jest`
- Updates typescript definitions to be more strict
- Enables `noImplicitAny` in tsconfig.json

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -7,12 +7,14 @@
     "target": "es6",
     "jsx": "react",
     "skipLibCheck": true,
+    "noImplicitAny": true,
```
Comment (zlarsen) [inline tsconfig.json:10]:
```suggestion
```

`"strict": true` enables this. https://www.typescriptlang.org/docs/handbook/compiler-options.html

File: .npmignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 src
-example
\ No newline at end of file
+example
+test
```
Comment (zlarsen) [inline .npmignore:3]: Nit
```suggestion
test

```

**Review by zlarsen — APPROVED** [review]
Nice work @jwdotjs !

🚀 🔮

File: tsconfig.json (hunk 1 above)
Comment (jwdotjs) [reply tsconfig.json:10]: Ah, thanks! I will update this

**Pull request merged**